testutil/web_suite.go

```diff
@@ -0,0 +1,46 @@
+package testutil
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/suite"
+)
+
+// WebSuite Base Suite for web Handlers
+type WebSuite struct {
+	suite.Suite
+	E *gin.Engine
+}
+
+// LoadGin initialize gin API
+func (suite *WebSuite) LoadGin() *gin.RouterGroup {
```

Comment:
And here you create a new suite and used `suite` as the variable...  I'm confused -- if `suite` is ok, why was it changed to `m` in MongoSuite?